### ISSUE TITLE
Error: Failed to apply style on Code 1.94.0

### ISSUE THREAD

**Issue by gitiy1** [rakibdev/material-code#9]
Uncached Error  Error: ENOENT: no such file or directory, open '/usr/lib/code/out/vs/code/electron-sandbox/workbench/workbench.html'

and the command output is
```
.rw-r--r-- 1.2k root  9 10月 05:32  workbench.esm.html
.rw-r--r-- 8.9k root  9 10月 05:32  workbench.js
```

**Comment by gitiy1** [comment]
it seems like change

https://github.com/rakibdev/material-code/blob/7ebb2470d871c1e796a24725fdb633a9f008ccaf/extension.js#L14

to

```js
const workbenchFile = vscode.Uri.joinPath(vscodeRoot, 'out/vs/code/electron-sandbox/workbench/workbench.esm.html')
```

could solve this problem but i am not sure if didn't cause some minor problems by the new logic.

**Comment by rakibdev** [comment]
yes fixed in [v3.0.0](https://github.com/rakibdev/material-code/releases)
now should work.
